src/components/LogInComponent/LoginFormComponent.jsx

```diff
@@ -1,48 +1,99 @@
-import { useSelector } from 'react-redux';
-import languageSelector from '../../assets/languages/languageSelector';
+import { useSelector } from "react-redux";
+import languageSelector from "../../assets/languages/languageSelector";
+import { useState } from "react";
+import axios from "axios";
+import Cookies from "universal-cookie";
+import { toast, ToastContainer } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import { useDispatch } from "react-redux";
+import { setUser } from "../../store/actions/user.action";
 
-const LoginFormComponent = ({ setWhichForm }) => {
+const LoginFormComponent = ({ setWhichForm, closeModal }) => {
+  const cookies = new Cookies();
+  const dispatch = useDispatch();
+  const authUrl = process.env.REACT_APP_AUTH_URL;
   const language = useSelector(state=> state.languageReducer);
+  const [dataUser, setDataUser] = useState({
+    email: "",
+    password: ""
+  });
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      const { data } = await axios.post(`${authUrl}/local/login`, dataUser);
+      cookies.set("token", data.token);
+      cookies.set("name", data.data.name);
+      cookies.set("last_name", data.data.last_name);
+      cookies.set("email", data.data.email);
+      dispatch(setUser(data.data));
```

Comment:
```js
const { 
 data: {
   token,
   data: { name,  last_name, email }
 }
} = await axios.post(`${authUrl}/local/login`, dataUser);
```

Reply:
Resolved in a different manner.

Reply:
However you did it well and onelinear, great, the code aesthetic is a well practice that must be part of us.